miniwin/miniwin/src/miniwin_ddpalette.cpp

```diff
@@ -12,10 +21,25 @@ HRESULT DirectDrawPaletteImpl::GetCaps(LPDWORD lpdwCaps)
 
 HRESULT DirectDrawPaletteImpl::GetEntries(DWORD dwFlags, DWORD dwBase, DWORD dwNumEntries, LPPALETTEENTRY lpEntries)
 {
+	for (int i = dwBase; i < dwNumEntries; i++) {
+		lpEntries[i].peRed = m_palette->colors[i].r;
+		lpEntries[i].peGreen = m_palette->colors[i].g;
+		lpEntries[i].peBlue = m_palette->colors[i].b;
```

Comment:
Do we care about [`peFlags`](https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-paletteentry)?
```suggestion
		lpEntries[i].peBlue = m_palette->colors[i].b;
		lpEntries[i].peFlags = 0;
```

Reply:
```suggestion
		lpEntries[i].peBlue = m_palette->colors[i].b;
		lpEntries[i].peFlags = PC_NONE;
```